configs/components/rubygem-gettext.rb

```diff
@@ -1,6 +1,16 @@
 component "rubygem-gettext" do |pkg, settings, platform|
-  pkg.version "3.2.2"
-  pkg.md5sum "4cbb125f8d8206e9a8f3a90f6488e4da"
+  # Projects may define a :rubygem_gettext_version setting, or we use 3.2.2 by default:
+  version = settings[:rubygem_gettext_version] || '0.6.1'
```

Comment:
Did this intend to be `3.2.2` instead of `0.6.1`?

Reply:
ugh. I fixed that locally but forgot to push it. That is what I get putting up PRs before I have coffee in the morning.